invoice.go

```diff
@@ -47,11 +60,13 @@ type InvoiceUpcomingInvoiceItemParams struct {
 // For more details see https://stripe.com/docs/api#create_invoice, https://stripe.com/docs/api#update_invoice.
 type InvoiceParams struct {
 	Params              `form:"*"`
+	AutoAdvance         *bool    `form:"auto_advance"`
 	ApplicationFee      *int64   `form:"application_fee"`
 	Billing             *string  `form:"billing"`
 	Closed              *bool    `form:"closed"`
 	Customer            *string  `form:"customer"`
 	DaysUntilDue        *int64   `form:"days_until_due"`
+	DefaultSource       *string  `form:"default_source"`
```

Comment:
I notice that `DefaultSource` on invoice and subscription are marked as expandable. Should we support that?

Reply:
Nice catch, fixed! PTAL